app/views/handlers/individual_response.py

```diff
@@ -182,30 +205,199 @@ class IndividualResponseHowHandler(IndividualResponseHandler):
         },
     }
 
-    def __init__(self, schema, questionnaire_store, language, form_data, list_item_id):
+    def __init__(
+        self,
+        schema,
+        questionnaire_store,
+        language,
+        request_args,
+        form_data,
+        list_item_id,
+    ):
         super().__init__(
             self.block_definition,
             schema,
             questionnaire_store,
             language,
+            request_args,
             form_data,
             list_item_id,
         )
 
     def handle_get(self):
-        previous_location_url = url_for(
-            "individual_response.request_individual_response",
-            list_item_id=self._list_item_id,
-        )
+        if self._request_args.get("journey") == "change":
+            previous_location_url = url_for(
+                "individual_response.get_individual_response_change",
+                list_item_id=self._list_item_id,
+            )
+        else:
+            previous_location_url = url_for(
+                "individual_response.request_individual_response",
+                list_item_id=self._list_item_id,
+            )
 
         return render_template(
-            "individual_response/how",
+            "individual_response/question",
             language=self._language,
             content=self.get_context(),
             previous_location_url=previous_location_url,
+            show_contact_us_guidance=True,
+        )
+
+    def handle_post(self):
```

Comment:
Is this if statement required? I think `url_for` will skip a param pair if the value is `None`

Reply:
I didn't know that `url_for` skips `None` values, good to know. I will update all occurrences of this.